infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -239,9 +239,10 @@ EOT
       secured_variables : [
         "CLIENT_SECRET",
         "CLIENT_ID",
-        "ACCOUNT_ID"
+        "ACCOUNT_ID",
+        "WRITABLE_ACCESS_TOKEN"
```

Comment:
WRITABLE, by convention, so instead of defining a read set and a write set of params, we can use just one.

Reply:
seems in correct - you had `WRITABLE` as suffix